controllers/userStatus.js

```diff
@@ -0,0 +1,121 @@
+const userStatusModel = require("../models/userStatus");
+
+/**
+ * Deletes a new User Status
+ *
+ * @param req {object} - Express Request Object
+ * @param res {object} - Express Response Object
+ *
+ */
+
+const deleteUserStatus = async (req, res) => {
+  try {
+    const { userId } = req.params;
+    const deletedUserStatus = await userStatusModel.deleteUserStatus(userId);
+    if (deletedUserStatus.userStatusExisted) {
+      return res.status(200).json({
+        id: deletedUserStatus.id,
+        userId,
+        message: "UserStatus Deleted successfully.",
+      });
+    } else {
+      return res.status(404).json({
+        id: deletedUserStatus.id,
+        userId,
+        message: "Could not find the user status linked to the user for deletion.",
+      });
```

Comment:
this is becoming a bit repetitive, can you look at this, we are sending the same response mostly in both the places

Reply:
As Mahima is blocked on this PR hence will resolve the pending comments in a new PR.[ Issue ticket ](https://github.com/Real-Dev-Squad/website-backend/issues/834) for the same.

Reply:
Since it was not merged, resolved in the same PR.